test/Form_test.js

```diff
@@ -925,4 +925,63 @@ describe("Form", () => {
       });
     });
   });
+
+  describe("Attributes", () => {
+    const formProps = {
+      schema: {},
+      id: "test-form",
+      className: "test-class other-class",
+      name: "testName",
+      method: "post",
+      target: "_blank",
+      action: "/users/list",
+      autocomplete: "off",
+      enctype: "multipart/form-data",
+      acceptcharset: "ISO-8859-1"
+    };
+
+    const comp = renderIntoDocument(
+      <div>
+        <Form {...formProps} />
+      </div>
+    );
+
+    const node = findDOMNode(comp).querySelector("form");
```

Comment:
You need to put this in a `beforeEach` statement so it's evaluated. Also, please use the `createFormComponent` helper.